app/src/pages/authoring/index.tsx

```diff
@@ -65,6 +97,23 @@ export default function AuthoringPage() {
           <Button w={240} loading={draftMutation.isLoading} onClick={createResource}>
             {`Create New Draft ${resourceType}`}
           </Button>
+          <Title order={3}>{`Start From an Existing ${resourceType}:`}</Title>
+          {artifacts ? (
+            <Select
+              label={`Select an existing ${resourceType} to clone`}
+              data={artifacts}
+              value={selectedArtifact}
+              onChange={setSelectedArtifact}
+            />
+          ) : (
+            <Text>No artifacts</Text>
```

Comment:
Instead of rendering text, can we make use of the `isLoading` attribute of the tRPC query to either disable the dropdown, render a spinner for it instead, etc.?